modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -19,12 +20,12 @@ Accessing Aura with SSO requires:
 
 Console SSO allows you to log in to the Aura Console using company IdP credentials and grants link:{neo4j-docs-base-uri}/cypher-manual/current/administration/access-control/built-in-roles#access-control-built-in-roles-public[Public Access privileges] to all instances in the tenant.
 
-The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported for Console-level Authentication:
+The following OpenID connect (OIDC) certified Identity Providers (IdPs) are currently supported for Console-level Authentication with Aura Enterprise and Aura Business Critical:
```

Comment:
Why is this lower case? Since it's an abbreviation, it's ok to camel cap if needed.